app/views/includes/_footer.njk

```diff
@@ -2,10 +2,16 @@
   <div class="nhsuk-footer" id="nhsuk-footer">
     <div class="nhsuk-width-container">
       <h2 class="nhsuk-u-visually-hidden">Support links</h2>
-      <ul class="nhsuk-footer__list">
-        <li class="nhsuk-footer__list-item"><a class="nhsuk-footer__list-item-link" href="https://aalto.digital.nhs.uk">Aalto</a></li>
-        <li class="nhsuk-footer__list-item"><a class="nhsuk-footer__list-item-link" href="https://aalto.digital.nhs.uk/#/integrated-apps?title=Reporting&url=https:~2F~2Fapp.powerbi.com~2Fview%3Fr%3DeyJrIjoiZjU0NTliMmUtZmUzOC00MjExLTk5ODQtNmUwNjhmNDMzNDcxIiwidCI6IjUwZjYwNzFmLWJiZmUtNDAxYS04ODAzLTY3Mzc0OGU2MjllMiIsImMiOjh9&library=5464c07f-daf1-4eee-b9b6-22e6c4dfbbd0">Portfolio Management</a></li>
-        </ul>
+      <ul class="nhsuk-footer__list nhsuk-footer__list--three-columns">
```

Comment:
These footer links are for patient facing services, we should check with @sinclr4 what links he wants in the footer

Reply:
Please leave the links as they currently are, there should only be the 2.